hbase_sql/src/main/java/com/splicemachine/olap/OlapServerMaster.java

```diff
@@ -96,6 +97,11 @@ private void run() throws Exception {
         // Initialize clients to ResourceManager and NodeManagers
         Configuration conf = HConfiguration.unwrapDelegate();
 
+        UserGroupInformation.isSecurityEnabled();
```

Comment:
Why check if security is enabled but not look at the result?

Reply:
That method has some side effects, that's what we care about.